src/field_theory/finite/trace.lean

```diff
@@ -18,10 +18,14 @@ finite field, trace
 
 namespace finite_field
 
+
 /-- The trace map from a finite field to its prime field is nongedenerate. -/
 lemma trace_to_zmod_nondegenerate (F : Type*) [field F] [finite F] {a : F}
```

Comment:
I suspect this lemma would be easier to use as
```suggestion
lemma trace_to_zmod_nondegenerate (F : Type*) [field F] [finite F] [algebra (zmod (ring_char F)) F] {a : F}
```
```

Reply:
Oh, nice!